### ISSUE TITLE
Error: `is_in_node_range` nil value

### ISSUE THREAD

**Issue by aspeddro** [monkoose/matchparen.nvim#16]
```
Error detected while processing CursorMoved Autocommands for "*":
Error executing lua callback: ...cker/start/matchparen.nvim/lua/matchparen/treesitter.lua:36: attempt to call field 'is_in_node_range' (a nil value)
stack traceback:
        ...cker/start/matchparen.nvim/lua/matchparen/treesitter.lua:36: in function 'get_skip_node'
        ...cker/start/matchparen.nvim/lua/matchparen/treesitter.lua:130: in function 'skip_by_region'
        ...k/packer/start/matchparen.nvim/lua/matchparen/search.lua:159: in function 'match_pos'
        ...acker/start/matchparen.nvim/lua/matchparen/highlight.lua:72: in function 'update'
        ...ite/pack/packer/start/matchparen.nvim/lua/matchparen.lua:60: in function <...ite/pack/packer/start/matchparen.nvim/lua/matchparen.lua:60>
```

```
NVIM v0.7.2
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by builduser

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

**Comment by monkoose** [comment]
Thank you. Will revert this, forgot it available only on 0.8 version.

**Comment by monkoose** [comment]
Done.
